cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -119,10 +118,10 @@
         });
     }
 
-    &__bottom-rule {
+    &.m-image-text-50-50__bottom-rule {
```

Comment:
Looks like this should probably follow the 25/75 format by including a div with the rule atom: https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html#L59-L61